cmd/revision.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/databus23/helm-diff/diff"
+	"github.com/databus23/helm-diff/manifest"
+	"github.com/mgutz/ansi"
+	"github.com/spf13/cobra"
+	"k8s.io/helm/pkg/helm"
+)
+
+const revisionCmdLongUsage = `
+This command compares the manifests details of a named release.
+
+It can be used to compare the manifests of 
+ 
+ - lastest REVISION with specified REVISION
+	$ helm diff revision [flags] [RELEASE] [REVISION]
+
+ - REVISION1 with REVISION2
+	$ helm diff revision [flags] [RELEASE] [REVISION1] [REVISION2]
+`
+
+func revisionCmd() *cobra.Command {
+	diff := diffCmd{}
+	revisionCmd := &cobra.Command{
+		Use:   "revision [flags] [RELEASE] REVISION1 REVISION2",
+		Short: "Shows diff between revision's manifests",
+		Long:  revisionCmdLongUsage,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if v, _ := cmd.Flags().GetBool("version"); v {
+				fmt.Println(Version)
+				return nil
+			}
+
+			switch {
+			case len(args) < 2:
+				return errors.New("Too few arguments to Command \"revision\".\nMinimum 2 arguments required: release name, revision")
+			case len(args) > 3:
+				return errors.New("Too many arguments to Command \"revision\".\nMaximum 3 arguments allowed: release name, revision1, revision2")
+			}
+
+			if q, _ := cmd.Flags().GetBool("suppress-secrets"); q {
```

Comment:
Maybe we can move this to a global flag?

Reply:
In my point of view, for executing `suppress-secrets` as a global pre-run , we might need to have a common diff object across all the subcommands and it must be made available in root.go

Reply:
opinions on this..? can this be done in some other way..?

Reply:
Hmm, you are right. Haven't thought of that, maybe leave it as it is for now. I think its more important to have the command cleanly separated. We can deduplicate later.